analysis/tvla.py

```diff
@@ -261,14 +261,14 @@ def run_tvla(ctx: typer.Context):
         rnd_list = [0]
         byte_list = [0]
     else:
-        if cfg["round_select"] is None:
+        if not cfg["round_select"]:
```

Comment:
As discussed offline, this change is necessary because `round_select` is now a list instead of an int. To find out if the list is empty (no `round_select`) specified, we have to use `not`.